src/pcapng.h

```diff
@@ -153,20 +158,39 @@ PcapNgWriteEnhancedPacket(
     long InterfaceId,
     long IsSend,
     long TimeStampHigh, // usec (unless if_tsresol is used)
-    long TimeStampLow
+    long TimeStampLow,
+    ULONG ProcessID
     )
 {
     int Err = NO_ERROR;
     struct PCAPNG_BLOCK_HEAD Head;
     struct PCAPNG_ENHANCED_PACKET_BODY Body;
     struct PCAPNG_BLOCK_OPTION_ENDOFOPT EndOption;
     struct PCAPNG_BLOCK_OPTION_EPB_FLAGS EpbFlagsOption;
+    struct PCAPNG_BLOCK_OPTION_COMMENT CommentOption;
     struct PCAPNG_BLOCK_TAIL Tail;
     char Pad[4] = {0};
+// COMMENT_MAX_SIZE must be multiple of 4
+#define COMMENT_MAX_SIZE 16
+    char Comment[COMMENT_MAX_SIZE] = { 0 };
```

Comment:
Remove this redundant zero-initialization since memset(0) is called below. This seems better than removing the memset because in the future the buffer might be dynamically sized.